release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3322,6 +3322,35 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+/ 4.14.29
+[id="ocp-4-14-29"]
+=== RHBA-2024:3697 - {product-title} 4.14.29 bug fix update and security update
+
+Issued: 2024-06-13
```

Comment:
```suggestion
Issued: 2024-06-11
```
Looks like this one shipped live yesterday?

Reply:
Unfortunetly, the release slipped byt a day because of a CVE issue. The following adviosires were shipped on the 13th:

- https://errata.devel.redhat.com/advisory/133125
- https://errata.devel.redhat.com/advisory/133127

Reply:
![Screenshot from 2024-06-13 12-43-11](https://github.com/openshift/openshift-docs/assets/57954076/36793144-f099-4cd8-828b-4f3640ab9a35)

Reply:
Argh, they didn't change the date on the customer advisory.